cypress/e2e/cloud/annotations.test.ts

```diff
@@ -257,13 +263,10 @@ describe('The Annotations UI functionality', () => {
     cy.getByTestID('button')
       .click()
       .then(() => {
-        cy.getByTestID('selector-list schmucket')
-          .click()
-          .getByTestID(`selector-list m`)
-          .click()
-          .getByTestID('selector-list v')
-          .click()
-          .getByTestID(`selector-list tv1`)
+        cy.getByTestID('selector-list schmucket').click()
+        cy.getByTestID(`selector-list m`).click()
+        cy.getByTestID('selector-list v').click()
+        cy.getByTestID(`selector-list tv1`)
```

Comment:
@genehynson can you explain to @ChitlangeSahas and me why this fixes the flakiness?

Reply:
Yes! So cypress generally isn't a fan of lost of chaining. The issue is that React updates the dom a lot and if the dom is updated in the middle of one of these long chains, cypress will lose its grasp on the element and it will error out saying "element was detached from the dom". My understanding is that basically whenever you chain commands, cypress interprets that as "use your reference to the dom from earlier _and_ do xyz". When you split them apart cypress always grabs the elements from the latest dom and things are less likely to be detached.

Reply:
btw to test these changes I ran each of these tests 20 times in a row. Before the change they were flaking out every 1 in 5 runs or so. With these changes all 20 runs passed